src/CommentionsServiceProvider.php

```diff
@@ -27,15 +28,18 @@ public function configurePackage(Package $package): void
             ->name(static::$name)
             ->hasConfigFile()
             ->hasViews()
-            ->hasMigration('create_commention_tables');
+            ->hasMigrations([
+                'create_commentions_tables',
+                'create_commentions_reactions_table',
+            ]);
     }
 
     public function packageBooted(): void
     {
         Livewire::component('commentions::comment', Comment::class);
         Livewire::component('commentions::comment-list', CommentList::class);
         Livewire::component('commentions::comments', Comments::class);
-
+        Livewire::component('commentions::reaction-manager', ReactionManager::class);
```

Comment:
Given the simplicity of the other component names should we just go with `Reactions::class` and `commentions::reactions`?

Reply:
Yeah I think that's a rename that makes sense